pkg/minikube/tunnel/tunnel.go

```diff
@@ -136,17 +141,53 @@ func setupRoute(t *tunnel) {
 
 	if !exists && len(conflict) == 0 {
 		t.status.RouteError = t.router.EnsureRouteIsAdded(t.status.TunnelID.Route)
-		if t.status.RouteError == nil {
-			//the route was added successfully, we need to make sure the registry has it too
-			//this might fail in race conditions, when another process created this tunnel
-			if err := t.registry.Register(&t.status.TunnelID); err != nil {
-				glog.Errorf("failed to register tunnel: %s", err)
-				t.status.RouteError = err
+		if t.status.RouteError != nil {
+			return
+		}
+		//the route was added successfully, we need to make sure the registry has it too
+		//this might fail in race conditions, when another process created this tunnel
+		if err := t.registry.Register(&t.status.TunnelID); err != nil {
+			glog.Errorf("failed to register tunnel: %s", err)
+			t.status.RouteError = err
+			return
+		} else if h.DriverName == "hyperkit" {
+			//the virtio-net interface acts up with ip tunnels :(
+			command := exec.Command("ifconfig", "bridge100")
+			fmt.Printf("About to run command: %s\n", command.Args)
+			response, _ := command.CombinedOutput()
+			iface := string(response)
+			pattern := regexp.MustCompile(`.*member: (en\d+) flags=3<LEARNING,DISCOVER>.*`)
```

Comment:
If your going to use "Must*", make it a package var.

Also, including flag values makes this regexp very fragile and susceptible to OS update churn. In lieu of someone making SCNetworkConfiguration available in Go, I wonder if it's possible to get the ifconfig, ipconfig, or networksetup commands to more simply list member interfaces. 

If not, the regexp approach is probably OK, but I think `member: en\d+` might be enough.